backend_plugin/src/main/evaluators/Evaluator.java

```diff
@@ -29,7 +29,7 @@
 	private RemoveImportEvaluator removeImportEval;
 
 	private IDocument document;
-	private DocumentChangesTracker documentChangesTracker;
+	private DocumentChangesListener documentChangesTracker;
```

Comment:
I'd recommend changing the variable name to documentChangesListener here. Keep it consistent throughout.

Reply:
Good call, I've updated the PR to reflect this.